src/deprecation.js

```diff
@@ -918,6 +918,24 @@ Object.defineProperty(core.utils, '_saidHello', {
 });
 
 /**
+ * Adds hooks for finding and uploading items.
+ *
+ * @param {Function} [addHook] - Function call that takes two parameters: `item:*, queue:Array`
+          function must return `true` if it was able to add item to the queue.
+ * @param {Function} [uploadHook] - Function call that takes two parameters: `prepare:CanvasPrepare, item:*` and
+ *        function must return `true` if it was able to handle upload of item.
+ * @return {PIXI.CanvasPrepare} Instance of plugin for chaining.
+ * @deprecated since 4.4.1
+ */
+prepare.BasePrepare.prototype.register = function register(addHook, uploadHook)
+{
+    warn('renderer.plugins.prepare.register is now deprecated, '
+        + 'please use renderer.plugins.prepare.registerFindHook & renderer.plugins.prepare.registerUploadHook');
+
+    return this.registerFindHook(addHook).registerUploadHook(uploadHook);
```

Comment:
Sure no problemo, both methods check internally but better safe than sorry. I should probably return `this` though to preserve chaining